data-prepper-plugins/s3-source/src/main/java/org/opensearch/dataprepper/plugins/source/s3/SqsWorker.java

```diff
@@ -53,6 +53,7 @@ public class SqsWorker implements Runnable {
     static final String SQS_MESSAGES_DELETED_METRIC_NAME = "sqsMessagesDeleted";
     static final String SQS_MESSAGES_FAILED_METRIC_NAME = "sqsMessagesFailed";
     static final String SQS_MESSAGES_DELETE_FAILED_METRIC_NAME = "sqsMessagesDeleteFailed";
+    static final String ZERO_SIZED_OBJECT_COUNT_METRIC_NAME = "zeroSizedObject";
```

Comment:
Everything looks good to me except the metrics name. All the s3 metrics starts with `s3` or `s3Object`. It can be `s3ObjectsWithSizeZero` or `s3Zero SizedObjects` or if you have any other suggestions.

Or we already have `s3ObjectSizeBytes` metric, so it can be `s3ObjectSizeZeroBytes`.